docs/handbook/text-anchors.rst

```diff
@@ -0,0 +1,140 @@
+
+.. _text-anchors:
+
+Text anchors
+============
+
+The ``anchor`` parameter determines alignment of drawn text relative to the ``xy`` parameter.
+The default alignment is top left, specifically ``la`` (left--ascender) for horizontal text
+and ``lt`` (left--top) for vertical text.
+
+This parameter is only supported by OpenType/TrueType fonts.
+Other fonts may ignore the parameter and use the default (top left) alignment.
+
+Specifying an anchor
+^^^^^^^^^^^^^^^^^^^^
+
+An anchor is specified with a two character string. The first character is the
+horizontal alignment, the second character is the vertical alignment.
+For example, the default value of ``la`` for horizontal text means left--ascender
+aligned text.
+
+When drawing text with :py:meth:`PIL.ImageDraw.ImageDraw.text` with a specific anchor,
+text will be placed such that the specified anchor point is at the ``xy`` coordinates.
+
+For example, in the following image, text is ``ms`` (middle--baseline) aligned, with ``xy`` at
+the intersection of the two lines:
+
+.. image:: ../../Tests/images/test_anchor_quick_ms.png
+  :alt: Middle--baseline aligned text.
+  :align: left
+
+.. code-block:: python
+
+    from PIL import Image, ImageDraw, ImageFont
+
+    font = ImageFont.truetype("Tests/fonts/NotoSans-Regular.ttf", 48)
+    im = Image.new("RGB", (200, 200), "white")
+    d = ImageDraw.Draw(im)
+    d.line(((0, 100), (200, 100)), "gray")
+    d.line(((100, 0), (100, 200)), "gray")
+    d.text((100, 100), "Quick", fill="black", anchor="ms", font=font)
+
+.. container:: clearer
+
+    |
+
+.. only: comment
+    The container above prevents the image alignment from affecting following text.
+
+Quick reference
+^^^^^^^^^^^^^^^
+
+.. image:: ../resources/anchor_horizontal.svg
+  :alt: Horizontal text
+  :align: center
+
+.. image:: ../resources/anchor_vertical.svg
+  :alt: Vertical text
+  :align: center
+
+Horizontal anchor alignment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``l`` --- left
```

Comment:
Nit: and for these triple dashes?

Reply:
1) A colon or shorter dash made the description too close and a little harder to read.
2) According to most style guides, the em-dash is an alternative to the colon, see https://en.wikipedia.org/wiki/Dash#Usage_2

(side note: `--` is an en-dash and `---` is an em-dash with Sphinx by default: https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=em-dash#confval-smartquotes_action)